WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -2985,20 +2983,19 @@ private List<Uri> convertStringArrayIntoUrisList(String[] stringArray) {
 
     private void addLastTakenPicture() {
         try {
-            // TODO why do we scan the file twice? Also how come it can result in OOM?
             WPMediaUtils.scanMediaFile(this, mMediaCapturePath);
             File f = new File(mMediaCapturePath);
             Uri capturedImageUri = Uri.fromFile(f);
             if (capturedImageUri != null) {
                 mEditorMedia.addNewMediaToEditorAsync(capturedImageUri, true);
-                final Intent scanIntent = new Intent(Intent.ACTION_MEDIA_SCANNER_SCAN_FILE);
-                scanIntent.setData(capturedImageUri);
-                sendBroadcast(scanIntent);
+                mEditorTracker.trackAddMediaFromDevice(mSite, true, false, capturedImageUri);
```

Comment:
⚠️  **Track issue**: I added an image to the image block by taking a photo. The `editor_photo_added` event was tracked twice. Calling `trackAddMediaFromDevice()` in `addLastTakenPicture` should be redundant.